script.js

```diff
@@ -1,18 +1,21 @@
 
 //start the game 
 let PlayerSelection = 'rock';
+let possibleChoices = ['rock', 'paper', 'scissors'];
+
     //computer chooses between rock, paper, scissors
 function getComputerChoice() {
+    debugger;
```

Comment:
So you shouldn't keep a debugger in your final code, but I added it in because it is so helpful. If you have a debugger set like this, and you reload your html in the brower with the inspector open, like this....
<img width="1789" alt="image" src="https://github.com/KinkPolice/janken/assets/7478250/576f1b31-9537-4b75-abdf-25aa77d517b7">

It'll pop you over to a tab where you can see your javascript and walk through it line by line. I use this nearly every day
<img width="1787" alt="image" src="https://github.com/KinkPolice/janken/assets/7478250/3c5cffdd-e6ea-4e02-bee1-d7a0c7170f0a">
So the screenshot below, here's after I walked past your "getChoice" variable. I can see what it's actually set to
<img width="1218" alt="image" src="https://github.com/KinkPolice/janken/assets/7478250/1dffe88b-7275-4c7f-aca9-f48869896d92">
And then here's when I walk a little farther and run into an error. It shows the line the error hit on and what type of error it is (assignment to a constant variable). Basically it's complaining that since `getChoice` wasn't declared like `let getChoice = 'whatever'` in the function, it doesn't want to change what it was assigned to originally (whatever `math.floor(math.random())` came out to). I forgot to fix that lol
<img width="871" alt="image" src="https://github.com/KinkPolice/janken/assets/7478250/4da4b9a1-0e92-4eff-a6a6-9fbf65439ca0">